src/pages/discovery/discovery.js

```diff
@@ -145,6 +147,7 @@ export default function Discovery() {
 				<DataTable
 					isSortIconResetNeeded={isSortIconResetNeeded}
 					setIsSortIconResetNeeded={setIsSortIconResetNeeded}
+					dataLoaded={dataLoaded}
```

Comment:
This will obviously need to be updated with the new variable name after that change is made.